db/migrate/20240610165140_update_dcyf_categories_and_eligibilties.rb

```diff
@@ -0,0 +1,285 @@
+      end
+
+      # Free City College
+      create_category 'Free City College'
+
+      # Learning English
+      rename_category from: 'Language', to: 'Learning English'
+
+      # Public Schools
+      create_category 'Public Schools'
+
+      # Reading & Literacy
+      rename_category from: 'Literacy Supports', to: 'Reading & Literacy'
+
+      # Child Welfare Services
+      create_category 'Child Welfare Services'
+
+      # Foster Care Services
+      rename_category from: 'Foster Care', to: 'Foster Care Services'
+
+      # Housing & Rental Assistance
+      rename_category from: 'Housing', to: 'Housing & Rental Assistance'
+
+      ['Housing Assistance', 'Emergency Rental Assistance'].each do |from|
+        migrate_resources_and_services_to_new_category from: from, to: 'Housing & Rental Assistance'
+      end
+
+      # Immigration Assistance
+      create_category 'Immigration Assistance'
+
+      migrate_resources_and_services_to_new_category from: 'Citizenship & Immigration', to: 'Immigration Assistance'
+
+      # Teen Parents
+      create_category 'Teen Parents'
+
+      # Crisis Intervention
+      create_category 'Crisis Intervention'
+
+      ['Suicide', 'Sexual Assault Hotlines', 'Hotlines and Case Management', 'Help Hotlines', 'Domestic Violence Hotlines'].each do |from|
+        migrate_resources_and_services_to_new_category from: from, to: 'Crisis Intervention'
+      end
+
+      # Gardening
+      rename_category from: 'Nature & Gardening', to: 'Gardening'
+
+      # Physical Fitness
+      create_category 'Physical Fitness'
+
+      ['Fitness & Exercise', 'Fitness/Exercise'].each do |from|
+        migrate_resources_and_services_to_new_category from: from, to: 'Physical Fitness'
+        delete_category from
+      end
+
+      # Team Sports
+      create_category 'Team Sports'
+
+      ['Basketball', 'Cheer', 'Dance', 'Football', 'Rec Teams', 'Soccer', 'Sports', 'Surfing', 'Swimming', 'Ultimate Frisbee'].each do |from|
+        migrate_resources_and_services_to_new_category from: from, to: 'Team Sports'
+        delete_category from
+      end
+
+      # Apprenticeship
+      create_category 'Apprenticeship'
+
+      # Career Exploration
+      rename_category from: 'Career Awareness', to: 'Career Exploration'
+
+      # Youth Jobs & Internships
+      rename_category from: 'Internships', to: 'Youth Jobs & Internships'
+
+
+      ## Eligibilities
+
+      # Hispanic/Latinx
+      rename_eligibility from: 'Latinx', to: 'Hispanic/Latinx'
+
+      # Middle School
+      rename_eligibility from: 'Middle School Students', to: 'Middle School'
+
+      # High School
+      rename_eligibility from: 'High School Students', to: 'High School'
+
+      # College
+      rename_eligibility from: 'College Students', to: 'College'
+
+      # CSF
+      create_eligibility 'CCSF'
+
+      # Experiencing Homelessness
+      rename_eligibility from: 'I am someone experiencing homelessness', to: 'Experiencing Homelessness'
```

Comment:
While getting the answers to the previous question, I also learned that we need to migrate all the services and organizations from the eligibility named `Homeless` into this new `Experiencing Homelessness` tag.

Reply:
Fixed in https://github.com/ShelterTechSF/askdarcel-api/pull/744/commits/30e74a9ffe6cc6fd996be1e0ca675a52fcb40e6c